flake.nix

```diff
@@ -2,307 +2,318 @@
-      milkv-pioneer = import ./milkv/pioneer;
-      morefine-m600 = import ./morefine/m600;
-      msi-b350-tomahawk = import ./msi/b350-tomahawk;
-      msi-b550-a-pro = import ./msi/b550-a-pro;
-      msi-gs60 = import ./msi/gs60;
-      msi-gl62 = import ./msi/gl62;
-      nxp-imx8mp-evk = import ./nxp/imx8mp-evk;
-      nxp-imx8mq-evk = import ./nxp/imx8mq-evk;
-      nxp-imx8qm-mek = import ./nxp/imx8qm-mek;
-      hardkernel-odroid-hc4 = import ./hardkernel/odroid-hc4;
-      hardkernel-odroid-h3 = import ./hardkernel/odroid-h3;
-      omen-14-fb0798ng = import ./omen/14-fb0798ng;
-      omen-15-en0010ca = import ./omen/15-en0010ca;
-      omen-16-n0005ne = import ./omen/16-n0005ne;
-      omen-16-n0280nd = import ./omen/16-n0280nd;
-      omen-15-en1007sa = import ./omen/15-en1007sa;
-      omen-15-en0002np = import ./omen/15-en0002np;
-      onenetbook-4 = import ./onenetbook/4;
-      olimex-teres_i = import ./olimex/teres_i;
-      pcengines-apu = import ./pcengines/apu;
-      pine64-pinebook-pro = import ./pine64/pinebook-pro;
-      pine64-rockpro64 = import ./pine64/rockpro64;
-      pine64-star64 = import ./pine64/star64;
-      protectli-vp4670 = import ./protectli/vp4670;
-      purism-librem-13v3 = import ./purism/librem/13v3;
-      purism-librem-15v3 = import ./purism/librem/15v3;
-      purism-librem-5r4 = import ./purism/librem/5r4;
-      raspberry-pi-2 = import ./raspberry-pi/2;
-      raspberry-pi-3 = import ./raspberry-pi/3;
-      raspberry-pi-4 = import ./raspberry-pi/4;
-      raspberry-pi-5 = import ./raspberry-pi/5;
-      kobol-helios4 = import ./kobol/helios4;
-      samsung-np900x3c = import ./samsung/np900x3c;
-      starfive-visionfive-v1 = import ./starfive/visionfive/v1;
-      starfive-visionfive-2 = import ./starfive/visionfive/v2;
-      supermicro = import ./supermicro;
-      supermicro-a1sri-2758f = import ./supermicro/a1sri-2758f;
-      supermicro-m11sdv-8c-ln4f = import ./supermicro/m11sdv-8c-ln4f;
-      supermicro-x10sll-f = import ./supermicro/x10sll-f;
-      supermicro-x12scz-tln4f = import ./supermicro/x12scz-tln4f;
-      system76 = import ./system76;
-      system76-gaze18 = import ./system76/gaze18;
-      system76-darp6 = import ./system76/darp6;
-      toshiba-swanky = import ./toshiba/swanky;
-      tuxedo-infinitybook-v4 = import ./tuxedo/infinitybook/v4;
-      tuxedo-infinitybook-pro14-gen7 = import ./tuxedo/infinitybook/pro14/gen7;
-      tuxedo-pulse-14-gen3 = import ./tuxedo/pulse/14/gen3;
-      tuxedo-pulse-15-gen2 = import ./tuxedo/pulse/15/gen2;
+    nixosModules =
+      let
+        deprecated =
+          issue: name: value:
+          builtins.trace "warning: ${name} flake output is deprecated and will be removed. See https://github.com/NixOS/nixos-hardware/issues/${issue} for more information" value;
+        import = path: path; # let the module system know what we are exporting
+      in
+      {
+        acer-aspire-4810t = import ./acer/aspire/4810t;
```

Comment:
What happend in this file?

Reply:
@Mic92 Oops, looks like my formatter from Doom Emacs did that. Since I'm not seeing any formatting guidelines in the contributers file, is it okay to leave it the way it is in the commit or would you like me to switch it back to the way it is in master?

Reply:
Sounds good, I just switched back to the nixos-hardware master format for flake.nix in the latest commit.